### ISSUE TITLE
Keithley Multimeter

### ISSUE THREAD

**Issue by ChrisM-S** [ISISComputingGroup/IBEX#3637]
As a scientist I would like to be able to easily set up the Keithley 2400 in a mode which allows me to use my own custom python script for Thermometry control.  This would need the default keithley settings to be changed to a quiescent mode, rather than scanning at the same time as my script is running and upsetting things.

**Comment by kjwoodsISIS** [comment]
@ChrisM-S - who is this for?

**Comment by ChrisM-S** [comment]
This is something to address the issue we had on Friday afternoon (on POLARIS) where the Keithley IOC was over-controlling and interfering with the users python scripting control for the same experiment.  The setup for the experiment had run successfully previously and involved work from us and the SE Electronics group.  I believe that it was having updated with the standard Keithly IOC on the last IBEX rollout which introduced (or re-introduced) the scanning into the db record for the IOC.

This is essentially a configuration management problem which we need to address (there is a lost time report on this one hence why I've framed this as a story from the scientist/user).

**Comment by kjwoodsISIS** [comment]
@Tom-Willemsen - is this ticket still required?

**Comment by ChrisM-S** [comment]
As long as it's **not** going to happen again there is no need for this ticket.  For me, it would be important that (on POLARIS at least) there is a configuration for the Electro-cell experiment they can and know to select which ensures passive operation from the IOC.  It might also be necessary for a quiescent mode check box on the OPI?  Perhaps the user script should only communicate through a block and this needs more thought?

Does anyone else see how we prevent this happening again.
   a) on POLARIS?
   b) on other instruments?

**Comment by Tom-Willemsen** [comment]
Yes, I do see this experiment happening again (specifically, on the 5th March!)

**Comment by kjwoodsISIS** [comment]
Put ticket back on the backlog until we are ready to do it.
